pom.xml

```diff
@@ -69,25 +69,10 @@
   </licenses>
 
   <modules>
-    <module>ratis-docs</module>
     <module>ratis-proto</module>
```

Comment:
Maybe `ratis-proto` can be deleted after moving `Hadoop.proto` to `ratis-hadoop` module?  It needs some tweaks in `ratis-hadoop/pom.xml`, too.

Reply:
POC: https://github.com/adoroszlai/ratis-hadoop-projects/commit/fb409d42881e846df112951b9d5ef50418223894

Reply:
Filed https://issues.apache.org/jira/browse/RATIS-1517 for the follow up change.  Since you have the POC, do you want to work on RATIS-1517?

Reply:
Sure, thanks.

Reply:
That's great!  Thanks a lot.